ProcessMaker/processmaker
# FOUR-13914 | The Language Selected for a User Is Not Complete

# Issue
Ticket: [FOUR-13914](https://processmaker.atlassian.net/browse/FOUR-13914)

When a language is selected for a user, all strings are not translated into the selected language.

# Solution
- Ensure missing translations are wrapped in the `$t(...)` translation method.
- Ensure missing translations are added to the language JSON files.

# How to Test
1. Go to branch `observation/FOUR-13914-b` in `processmaker`.
2. Go to branch `observation/FOUR-13914-b` in `screen-builder`.
3. Create a user. Change their language to `es` for Spanish.
4. Ensure that the text on the following pages is translated into Spanish:
	- Homepage (Custom Dashboard text)
	- Processes
	- Designer

ci:next

# Code Review Checklist


[FOUR-13914]: https://processmaker.atlassian.net/browse/FOUR-13914?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
